package.json

```diff
@@ -59,6 +59,8 @@
     "babel-plugin-transform-es2015-modules-commonjs": "6.8.0",
     "babel-plugin-transform-runtime": "6.9.0",
     "babel-preset-es2015-loose-rollup": "7.0.0",
+    "babel-register": "6.9.0",
```

Comment:
What version of node/npm do you have? I did a fresh install and both were installed for me.